enterprise/enterprise.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2018 Pilosa Corp. All rights reserved.
```

Comment:
I think we should add a doc.go to this package, or a godoc comment on this file explaining how the enterprise injection works